src/patchers/NodePatcher.js

```diff
@@ -1168,4 +1168,12 @@ export default class NodePatcher {
       receiver = receiver.parent;
     }
   }
+
+  /**
+   * Gets the first "interesting token" in the indexed range (default range is `this` + parent)
+   */
+  getFirstSemanticToken(from: number=this.contentStart, to: number=this.parent.contentEnd): ?SourceToken {
```

Comment:
I imagine the default end of the range may need to be tweaked in the future if this gets used more, since I think sometimes a node can have the same end as its parent. And it might be best to just have no end and just find the next token. But looks like this works fine at least for this case.